### ISSUE TITLE
Unable to Verify - no message received by TS client

### ISSUE THREAD

**Issue by Forge-Media** [Newcomer1989/TSN-Ranksystem#532]
Hi,

**Issue:** Updated TSN to v1.2.9 and noticed that users can no longer verify, as they receive no message from the bot. All [PHP extensions](https://i.imgur.com/He5K3bP.png) and commands are enabled.

Gif showing that the Verify webinterface can see the user in our Teamspeak's `Verify Channel` and when you selected the user in the drop-down list the page refreshes, however, no message is received by the client: [Link to GIF](https://i.imgur.com/OVtI9dG.gif)

Our Teamspeak 3 server is version: 3.3.0

**Logs:**
```
2018-08-19 19:18:09.282023   INFO      Bot starts now his work!
2018-08-19 19:18:09.281955   INFO      Config check [done]
2018-08-19 19:18:09.281069   INFO      Config check started...
2018-08-19 19:18:09.280945   INFO        Joined to specified Channel.
2018-08-19 19:18:08.288010   INFO        Connection to TS3 Server established.
2018-08-19 19:18:07.429741   INFO      Connect to TS3 Server (Address: "xxx.xxx.xxx.xx" Voice-Port: "9987" Query-Port: "10011").
2018-08-19 19:18:07.429676   INFO      Loading addons [done]
2018-08-19 19:18:07.429464   INFO        Addon: 'assign_groups' [ON]
2018-08-19 19:18:07.429128   INFO      Loading addons...
2018-08-19 19:18:07.429074   INFO      Ranksystem Version: 1.2.9
2018-08-19 19:18:07.429019   INFO      Check Ranksystem files for updates [done]
2018-08-19 19:18:07.428958   INFO      Check Ranksystem files for updates...
2018-08-19 19:18:07.428883   INFO      Check Ranksystem database for updates [done]
2018-08-19 19:18:07.428793   INFO        Check files permissions [done]
2018-08-19 19:18:07.421964   INFO        Check files permissions...
2018-08-19 19:18:07.421567   INFO        No newer version detected; Database check finished.
2018-08-19 19:18:07.421508   INFO      Check Ranksystem database for updates...
2018-08-19 19:18:07.421450   DEBUG     Database Version: 5.7.23-log
2018-08-19 19:18:07.421386   DEBUG     Using PHP Version: 5.6.37
2018-08-19 19:18:07.421313   DEBUG     Running on OS: Linux 4.4.0-131-generic
2018-08-19 19:18:07.416299   INFO      Initialize Bot...
2018-08-19 19:18:07.416244   INFO      ###################################################################
2018-08-19 19:18:07.416185   INFO      
2018-08-19 19:18:07.416056   INFO      ###################################################################
2018-08-19 19:18:06.675666   INFO      Received signal to stop! Shutting down!
``` 

_TSN dashboard is hosted on an Ubuntu 16.04.5 DigitalOcean VPS and is using https_

Any ideas what I can do to try solve this, the ranksystem.logs does not show any errors?
Thanks!

**Comment by Newcomer1989** [comment]
This issue comes with version 1.2.9.

It will be fixed with the next release 1.2.10. We hope, we can release it the next few days.

**Comment by Newcomer1989** [comment]
Version 1.2.10 is out now.
